pkg/cmd/cli/cmd/volume.go

```diff
@@ -35,29 +37,32 @@ just those that match a wildcard.`
   // The volume name is auto generated
   $ %[1]s volume dc/registry --add --mount-path=/opt
 
-  // Add new volume 'v1' with secret 'magic' for pod 'p1'
-  $ %[1]s volume pod/p1 --add --name=v1 -m /etc --type=secret --secret-name=magic
+  // Add new volume 'v1' with secret 'magic' for replication controller 'r1'
+  $ %[1]s volume rc/r1 --add --name=v1 -m /etc --type=secret --secret-name=magic
 
-  // Add new volume to pod 'p1' based on gitRepo (or other volume sources not supported by --type)
-  $ %[1]s volume pod/p1 --add -m /repo --source=<json-string>
+  // Add new volume to replication controller 'r1' based on gitRepo
+  // or other volume sources not supported by --type
+  $ %[1]s volume rc/r1 --add -m /repo --source=<json-string>
 
-  // Add emptyDir volume 'v1' to a pod definition on disk and update the pod on the server
-  $ %[1]s volume -f pod.json --add --name=v1
+  // Add emptyDir volume 'v1' to a deployment config definition on disk and 
+  // update the deployment config on the server
+  $ %[1]s volume -f dc.json --add --name=v1
 
   // Create a new persistent volume and overwrite existing volume 'v1' for replication controller 'r1'
   $ %[1]s volume rc/r1 --add --name=v1 -t persistentVolumeClaim --claim-name=pvc1 --overwrite
 
-  // Change pod 'p1' mount point to /data for volume v1
-  $ %[1]s volume pod p1 --add --name=v1 -m /data --overwrite
+  // Change replication controller 'r1' mount point to /data for volume v1
+  $ %[1]s volume rc r1 --add --name=v1 -m /data --overwrite
```

Comment:
Make it more clear that the "change" action happens beucause of `--overwrite`. Maybe something like `// Ovewrite the existing replication controller 'r1' mount point to /data for volume v1` or `// Change (ovewrite) replication controller 'r1' mount point to /data for volume v1`.